source/validate_id.cpp

```diff
@@ -1216,12 +1216,11 @@ bool idUsage::isValid<SpvOpFunctionParameter>(const spv_instruction_t* inst,
                                    << inst->words[resultTypeIndex]
                                    << "' is not defined.";
            return false);
-  auto function = inst - 1;
+  auto function = inst;
```

Comment:
This seems kind of pointless.  Just go ahead and rename the parameter inst to function if you like.